src/web/view/ScratchMode.re

```diff
@@ -246,50 +315,60 @@ module Selection = {
   open Cursor;
 
   [@deriving (show({with_path: false}), sexp, yojson)]
-  type t = CellEditor.Selection.t;
```

Comment:
i don't get why we need to reify TextBox selection here? why do we need to change this type and add the resulting logic?